src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator {
```
так всі наслідники цього класу будуть імплементити AreaCalculator автоматично